metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -409,6 +428,7 @@ chkconfig --del metron-management-ui
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 %changelog
+* Thu Jun 29 2017 Add Metron Management jar - 0.4.0+1
```

Comment:
Perhaps this should be:

```
* Thu Jun 29 2017 Otto Fowler <otto@apache.org> - 0.4.0+1
- Added Metron Management jar
```
Or similar?  Not clean on when we use individual names/contact info or `Apache Metron <dev@metron.apache.org>`.